### PR TITLE
fixes(#8500) Replace constants

### PR DESCRIPTION
**Pull request by cherish2003**
### Because

   - To replace constants
### This commit
   - Replaces NimbusConstants with NimbusExperiment

### CODE REVIEW

**Comment by cherish2003** [comment]
@yashikakhurana i have made the changes as required kindly review it and let me know any changes required

**Comment by cherish2003** [comment]
@yashikakhurana i have made changes as you told but i dont know why the test cases are failing

File: experimenter/experimenter/experiments/admin.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     NimbusBranchChangeLogSerializer,
```
Comment (yashikakhurana) [inline experimenter/experimenter/experiments/admin.py:13]: we don't need to change here

**Review by yashikakhurana — CHANGES_REQUESTED** [review]

File: experimenter/experimenter/experiments/admin.py (hunk 1 above)
Comment (cherish2003) [reply experimenter/experimenter/experiments/admin.py:13]: @yashikakhurana means no need to modify the changes in this file am i right ?

Comment (yashikakhurana) [reply experimenter/experimenter/experiments/admin.py:13]: yes

Comment (cherish2003) [reply experimenter/experimenter/experiments/admin.py]: @yashikakhurana once check out i have the changes in that file

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
@cherish2003 I just searched and there are bunch of other files where this can be changed, I will suggest to look into this more

**Comment by cherish2003** [comment]
@yashikakhurana can you specify the folders i dont know in which folders the changes are needed to be made

**Comment by yashikakhurana** [comment]
> @yashikakhurana can you specify the folders i dont know in which folders the changes are needed to be made
if you will search in the entire project, you will get the sense, but majorly in test files

**Comment by cherish2003** [comment]
@yashikakhurana i have made changes in all the files once have a look at it

**Comment by cherish2003** [comment]
@yashikakhurana mama once review the PR kindly inform about the changes

File: experimenter/experimenter/experiments/tests/test_migrations.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 from django_test_migrations.contrib.unittest_case import MigratorTestCase
 
-from experimenter.experiments.constants import NimbusConstants
```
Comment (yashikakhurana) [inline experimenter/experimenter/experiments/tests/test_migrations.py:3]: you can ignore changes in this file, that's why your ci is failing

File: experimenter/experimenter/experiments/api/v5/types.py [hunk 3]
```diff
@@ -11,7 +11,6 @@
     TransitionConstants,
```
Comment (yashikakhurana) [inline experimenter/experimenter/experiments/api/v5/types.py:11]: ignore this file too

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
just some files not to be changed, just tagged those files, thank you :)

**Comment by cherish2003** [comment]
@yashikakhurana thanks for the reply i will modify and push the changes

**Review by yashikakhurana — APPROVED** [review]
thank you @cherish2003 👏

**Pull request merged**